arch/TMS320C64x/TMS320C64xMapping.c

```diff
@@ -1718,17 +1718,156 @@ void TMS320C64x_get_insn_id(cs_struct *h, cs_insn *insn, unsigned int id)
 }
 
 #ifndef CAPSTONE_DIET
+//grep TMS320C64X_INS include/capstone/tms320c64x.h | awk '{print "{"$1 "\""tolower(substr($1, 16, length($1)-16))"\"""},"}'
 static name_map insn_name_maps[] = {
-	{ TMS320C64X_INS_INVALID, NULL },
-
-	{ TMS320C64X_INS_ADD, "add" },
-	{ TMS320C64X_INS_NOP, "nop" },
+	{TMS320C64X_INS_INVALID,NULL},
```

Comment:
could you put a space after the comma here, and also below?

Reply:
Like this ?

Reply:
yes, this is just about coding style